gusto/limiters.py

```diff
@@ -178,3 +178,35 @@ def apply(self, field):
                 applied, if this was not a blank limiter.
         """
         pass
+
+
+class MixedFSLimiter(object):
+    """An object to hold a dictionary that defines limiters for transported prognostic
+    variables. Different limiters may be applied to different fields and not every transported variable needs a defined limiter.
+    """
+
+    def __init__(self, equation, sublimiters):
+        """
+        Args:
+            equation (:class: 'PrognosticEquationSet'): the prognostic equation(s)
```

Comment:
```suggestion
            equation (:class:`PrognosticEquationSet`): the prognostic equation(s)
```
I think we need a back quotation here and not forward ones!